kubetest/extract_k8s.go

```diff
@@ -73,7 +73,7 @@ func (l *extractStrategies) String() string {
 func (l *extractStrategies) Set(value string) error {
 	var strategies = map[string]extractMode{
 		`^(local)`:                            local,
-		`^gke-?(default|latest(-\d+.\d+)?)?$`: gke,
+		`^gke-?(default|channel-(rapid|stable|release)|latest(-\d+.\d+)?)?$`: gke,
```

Comment:
it should be `regular` instead `release`

Reply:
done